test/media/manifest_parser_unit.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe('ManifestParser', () => {
+    describe('getMimeType', () => {
+        it('returns the correct mimeType', async () => {
+            const netEngine = new shaka.test.FakeNetworkingEngine()
+                .setHeaders('dummy://foo', { 'content-type' : 'application/dash+xml'});
+            const mimeType = await shaka.media.ManifestParser
+                .getMimeType('dummy://foo', netEngine, 3);
```

Comment:
The third argument is a `RetryParameters` object, not a number.  You should use `shaka.net.NetworkingEngine.defaultRetryParameters()`.